msal-python-conceptual/advanced/macos-broker.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using MSAL Python with Authentication Brokers on macOS
+description: "Using authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of advanced functionality such as token binding, protecting any issued tokens from exfiltration and misuse."
+author: localden
+manager: CelesteDG
+
+ms.service: msal
+ms.subservice: msal-python
+ms.topic: conceptual
+ms.date: 09/06/2024
+ms.author: ddelimarsky
+ms.reviewer: shermanouko, rayluo
+---
+
+# Using MSAL Python with Authentication Brokers on macOS
+
+>[!NOTE]
+>macOS authentication broker support is introduced with `msal` version 1.31.0.
+
+Using authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of advanced functionality such as **token binding**, protecting any issued tokens from exfiltration and misuse.
```

Comment:
Same as above, change brokers to broker, and change "protecting any issued tokens from exfiltration and misuse." to "protecting Entra ID refresh tokens".